internal/command/machine/run.go

```diff
@@ -186,25 +220,73 @@ func newRun() *cobra.Command {
 			Description: "Enable LSVD for this machine",
 			Hidden:      true,
 		},
+		flag.Bool{
+			Name:        "it",
```

Comment:
What about naming it `--interactive` or just removing it in favor `--shell`? they are different but most people just want what --shell provides.

Reply:
`-it` is a confusing flag. in docker it's just 2 flags sharing a dash. `interactive` and `tty`

Reply:
Killed this arg. It's just `--shell` now.